src/main/java/thePackmaster/cardmodifiers/madsciencepack/ApplyFrosbiteModifier.java

```diff
@@ -14,7 +15,7 @@ public ApplyFrosbiteModifier(int valueIn){
 
     @Override
     public String modifyDescription(String rawDescription, AbstractCard card) {
-        return rawDescription + CardCrawlGame.languagePack.getUIString("MadScienceModifiers").TEXT[1];
+        return rawDescription + CardCrawlGame.languagePack.getUIString(SpireAnniversary5Mod.makeID("MadScienceModifiers")).TEXT[1];
```

Comment:
All the changes like this are because there was an entry that wasn't prefixed with the mod ID.